### ISSUE TITLE
[Bug]: 24w14a Crashes

### ISSUE THREAD

**Issue by lowercasebtw** [TreyRuffy/BetterF3#75]
### Describe the Bug

In 24w14a, if you press F3 the game crashes with this error
```
[01:56:43] [Render thread/ERROR]: Unreported exception thrown!
java.lang.NoSuchMethodError: 'boolean net.minecraft.client.MinecraftClient.method_1540()'
	at me.cominixo.betterf3.modules.SystemModule.update(SystemModule.java:98) ~[BetterF3-10.0.0-alpha.1-Fabric-24w12a.jar:?]
	at me.cominixo.betterf3.utils.DebugRenderer.newText(DebugRenderer.java:218) ~[BetterF3-10.0.0-alpha.1-Fabric-24w12a.jar:?]
```

### Steps to Reproduce

Open 24w14a
Press F3
Crash

### Expected Behavior

Display BetterF3 Menu

### Screenshots

![image](https://github.com/TreyRuffy/BetterF3/assets/126462578/7e90b923-e93d-467c-9c30-aa8c35d1c9fc)
![image](https://github.com/TreyRuffy/BetterF3/assets/126462578/7162192a-0292-4d86-8314-85f41e3a115c)

### Modloader Type

Fabric

### Modloader Version

0.15.9

### Mod Version

10.0.0-alpha.1

### Log Output

```shell
[01:56:43] [Render thread/ERROR]: Unreported exception thrown!
java.lang.NoSuchMethodError: 'boolean net.minecraft.client.MinecraftClient.method_1540()'
	at me.cominixo.betterf3.modules.SystemModule.update(SystemModule.java:98) ~[BetterF3-10.0.0-alpha.1-Fabric-24w12a.jar:?]
	at me.cominixo.betterf3.utils.DebugRenderer.newText(DebugRenderer.java:218) ~[BetterF3-10.0.0-alpha.1-Fabric-24w12a.jar:?]
	at MC/net.minecraft.client.gui.hud.DebugHud.handler$zzj000$betterf3$drawRightText(DebugHud.java:1725) ~[client-intermediary.jar:?]
	at MC/net.minecraft.client.gui.hud.DebugHud.drawRightText(DebugHud.java) ~[client-intermediary.jar:?]
	at MC/net.minecraft.client.gui.hud.DebugHud.method_51746(DebugHud.java:141) ~[client-intermediary.jar:?]
	at MC/net.minecraft.client.gui.DrawContext.draw(DrawContext.java:91) ~[client-intermediary.jar:?]
	at MC/net.minecraft.client.gui.hud.DebugHud.render(DebugHud.java:139) ~[client-intermediary.jar:?]
	at MC/net.minecraft.client.gui.hud.InGameHud.method_55807(InGameHud.java:190) ~[client-intermediary.jar:?]
	at MC/net.minecraft.client.gui.LayeredDrawer.renderInternal(LayeredDrawer.java:34) ~[client-intermediary.jar:?]
	at MC/net.minecraft.client.gui.LayeredDrawer.method_55812(LayeredDrawer.java:20) ~[client-intermediary.jar:?]
	at MC/net.minecraft.client.gui.LayeredDrawer.renderInternal(LayeredDrawer.java:34) ~[client-intermediary.jar:?]
	at MC/net.minecraft.client.gui.LayeredDrawer.render(LayeredDrawer.java:27) ~[client-intermediary.jar:?]
	at MC/net.minecraft.client.gui.hud.InGameHud.render(InGameHud.java:217) ~[client-intermediary.jar:?]
	at MC/net.minecraft.client.render.GameRenderer.render(GameRenderer.java:901) ~[client-intermediary.jar:?]
	at MC/net.minecraft.client.MinecraftClient.render(MinecraftClient.java:1322) ~[client-intermediary.jar:?]
	at MC/net.minecraft.client.MinecraftClient.run(MinecraftClient.java:890) [client-intermediary.jar:?]
	at net.minecraft.client.main.Main.main(Main.java:265) [client-intermediary.jar:?]
	at net.fabricmc.loader.impl.game.minecraft.MinecraftGameProvider.launch(MinecraftGameProvider.java:470) [fabric-loader-0.15.9.jar:?]
	at net.fabricmc.loader.impl.launch.knot.Knot.launch(Knot.java:74) [fabric-loader-0.15.9.jar:?]
	at net.fabricmc.loader.impl.launch.knot.KnotClient.main(KnotClient.java:23) [fabric-loader-0.15.9.jar:?]
	at org.prismlauncher.launcher.impl.StandardLauncher.launch(StandardLauncher.java:87) [NewLaunch.jar:?]
	at org.prismlauncher.EntryPoint.listen(EntryPoint.java:129) [NewLaunch.jar:?]
	at org.prismlauncher.EntryPoint.main(EntryPoint.java:70) [NewLaunch.jar:?]
[01:56:43] [Render thread/INFO]: [voicechat] Clearing audio channels
---- Minecraft Crash Report ----
// Hey, that tickles! Hehehe!

Time: 2024-04-07 01:56:43
Description: Unexpected error

java.lang.NoSuchMethodError: 'boolean net.minecraft.client.MinecraftClient.method_1540()'
	at me.cominixo.betterf3.modules.SystemModule.update(SystemModule.java:98)
	at me.cominixo.betterf3.utils.DebugRenderer.newText(DebugRenderer.java:218)
	at MC//net.minecraft.client.gui.hud.DebugHud.handler$zzj000$betterf3$drawRightText(DebugHud.java:1725)
	at MC//net.minecraft.client.gui.hud.DebugHud.drawRightText(DebugHud.java)
	at MC//net.minecraft.client.gui.hud.DebugHud.method_51746(DebugHud.java:141)
	at MC//net.minecraft.client.gui.DrawContext.draw(DrawContext.java:91)
	at MC//net.minecraft.client.gui.hud.DebugHud.render(DebugHud.java:139)
	at MC//net.minecraft.client.gui.hud.InGameHud.method_55807(InGameHud.java:190)
	at MC//net.minecraft.client.gui.LayeredDrawer.renderInternal(LayeredDrawer.java:34)
	at MC//net.minecraft.client.gui.LayeredDrawer.method_55812(LayeredDrawer.java:20)
	at MC//net.minecraft.client.gui.LayeredDrawer.renderInternal(LayeredDrawer.java:34)
	at MC//net.minecraft.client.gui.LayeredDrawer.render(LayeredDrawer.java:27)
	at MC//net.minecraft.client.gui.hud.InGameHud.render(InGameHud.java:217)
	at MC//net.minecraft.client.render.GameRenderer.render(GameRenderer.java:901)
	at MC//net.minecraft.client.MinecraftClient.render(MinecraftClient.java:1322)
	at MC//net.minecraft.client.MinecraftClient.run(MinecraftClient.java:890)
	at net.minecraft.client.main.Main.main(Main.java:265)
	at app//net.fabricmc.loader.impl.game.minecraft.MinecraftGameProvider.launch(MinecraftGameProvider.java:470)
	at app//net.fabricmc.loader.impl.launch.knot.Knot.launch(Knot.java:74)
	at app//net.fabricmc.loader.impl.launch.knot.KnotClient.main(KnotClient.java:23)
	at app//org.prismlauncher.launcher.impl.StandardLauncher.launch(StandardLauncher.java:87)
	at app//org.prismlauncher.EntryPoint.listen(EntryPoint.java:129)
	at app//org.prismlauncher.EntryPoint.main(EntryPoint.java:70)


A detailed walkthrough of the error, its code path and all known details is as follows:
---------------------------------------------------------------------------------------

-- Head --
Thread: Render thread
Stacktrace:
	at me.cominixo.betterf3.modules.SystemModule.update(SystemModule.java:98)
	at me.cominixo.betterf3.utils.DebugRenderer.newText(DebugRenderer.java:218)
	at MC//net.minecraft.client.gui.hud.DebugHud.handler$zzj000$betterf3$drawRightText(DebugHud.java:1725)
	at MC//net.minecraft.client.gui.hud.DebugHud.drawRightText(DebugHud.java)
	at MC//net.minecraft.client.gui.hud.DebugHud.method_51746(DebugHud.java:141)
	at MC//net.minecraft.client.gui.DrawContext.draw(DrawContext.java:91)
	at MC//net.minecraft.client.gui.hud.DebugHud.render(DebugHud.java:139)
	at MC//net.minecraft.client.gui.hud.InGameHud.method_55807(InGameHud.java:190)
	at MC//net.minecraft.client.gui.LayeredDrawer.renderInternal(LayeredDrawer.java:34)
	at MC//net.minecraft.client.gui.LayeredDrawer.method_55812(LayeredDrawer.java:20)
	at MC//net.minecraft.client.gui.LayeredDrawer.renderInternal(LayeredDrawer.java:34)
	at MC//net.minecraft.client.gui.LayeredDrawer.render(LayeredDrawer.java:27)
	at MC//net.minecraft.client.gui.hud.InGameHud.render(InGameHud.java:217)
```

**Comment by lowercasebtw** [comment]
no wonder betterf3 crashes in 24w14a
its using MinecraftClient.is64bit
xD

24w14a+ Is now only 64 bit so Minecraft removed the method/field "is64bit" from the MinecraftClient class

**Comment by BetaKors** [comment]
this also happens in 1.20.5. was about to make another issue but it would just be a duplicate
i believe the fix is prob gonna be to remove all instances of is64bit

**Comment by lowercasebtw** [comment]
Turns out it’s a bit more, text rendering changed.

**Comment by BetaKors** [comment]
i see. is that why i am getting some FPS drops on the new version (#79)? also, i found a "fix" for this crash: remove all instances of the system module, making its update method not run

**Comment by TreyRuffy** [comment]
Fixed in 10.0.0
